proto-lens-arbitrary/package.yaml

```diff
@@ -15,7 +15,7 @@ extra-source-files:
 
 dependencies:
   - proto-lens >= 0.4 && < 0.8
-  - base >= 4.10 && < 4.17
+  - base >= 4.10 && < 4.19
   - bytestring >= 0.10 && < 0.12
   - containers >= 0.5 && < 0.7
   - text >= 1.2 && < 2.1
```

Comment:
text-2.1 should be allowed since users can upgrade to this version even if they don't use it as part of [ghc 9.8](https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history)